cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md

```diff
@@ -176,7 +176,7 @@ Resource ownership is an essential concept in libcudf. In short, an "owning" obj
 resource (such as device memory). It acquires that resource during construction and releases the
 resource in destruction ([RAII](https://en.cppreference.com/w/cpp/language/raii)). A "non-owning"
 object does not own resources. Any class in libcudf with the `*_view` suffix is non-owning. For more
-detail see the [`libcudf++` presentation.](https://docs.google.com/presentation/d/1zKzAtc1AWFKfMhiUlV5yRZxSiPLwsObxMlWRWz_f5hA/edit?usp=sharing)
+detail see the [`libcudf` presentation.](https://docs.google.com/presentation/d/1zKzAtc1AWFKfMhiUlV5yRZxSiPLwsObxMlWRWz_f5hA/edit?usp=sharing)
```

Comment:
To be fair, this one is the only one that could still stay as `libcudf++`. More importantly, these slides should probably be somewhere other than my Google Drive... or just remove the link all together. The content within this guide should supersede anything in those slides by this point.

Reply:
I’m going to leave it as-is. The presentation is titled libcudf++ but it’s about libcudf and I don’t want readers to be confused about the proper name of the library.